tests/unit/cocoon/test_utilities.py

```diff
@@ -1598,3 +1594,14 @@ def test_identify_cash_items(self, _, cash_flag, remove_cash_items, ground_truth
             ground_truth, list(dataframe["__currency_identifier_for_LUSID"])
         )
         self.assertEqual(mappings_ground_truth, mappings_test)
+
+    def test_strip_whitespace(self):
+        df_true = pd.DataFrame([["GBP", "GBP", "GBP  USD"],
+                                ["GBP", "GBP", "GBP"],
+                                ["GBP", "GBP", "GBP"]], columns=["a", "b", "c"])
+        df_test = pd.DataFrame([["GBP   ", "GBP", "GBP  USD"],
+                                ["   GBP  ", "   GBP", "GBP"],
+                                ["GBP   ", "GBP", "GBP   "]], columns=["a", "b", "c"])
+        df_test = strip_whitespace(df_test)
+        [self.assertEqual(list(df_true[col]), list(df_test[col])) for col in list(df_test.columns)]
+        pass
```

Comment:
Did you leave this "pass" line in for any particular reason? I don't see it serving any purpose